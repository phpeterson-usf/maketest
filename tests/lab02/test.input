$(TESTS_DIR)/test.ntc
