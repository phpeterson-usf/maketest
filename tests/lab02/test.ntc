# Calculation
x = 2
y = 4
z = ((100 * 2) / 4) + 1
# Print result
printstr("z = ")
printdec(z)
println()
