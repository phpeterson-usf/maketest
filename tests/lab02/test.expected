TK_IDENT("x")
TK_EQ("=")
TK_INTLIT("2")
TK_EOL("")
TK_IDENT("y")
TK_EQ("=")
TK_INTLIT("4")
TK_EOL("")
TK_IDENT("z")
TK_EQ("=")
TK_LPAREN("(")
TK_LPAREN("(")
TK_INTLIT("100")
TK_MULT("*")
TK_INTLIT("2")
TK_RPAREN(")")
TK_DIV("/")
TK_INTLIT("4")
TK_RPAREN(")")
TK_PLUS("+")
TK_INTLIT("1")
TK_EOL("")
TK_IDENT("printstr")
TK_LPAREN("(")
TK_STRLIT("z = ")
TK_RPAREN(")")
TK_EOL("")
TK_IDENT("printdec")
TK_LPAREN("(")
TK_IDENT("z")
TK_RPAREN(")")
TK_EOL("")
TK_IDENT("println")
TK_LPAREN("(")
TK_RPAREN(")")
TK_EOL("")
TK_EOT("")
